tests/Unit/Isomorphic/DeferredRequestRegistryTest.php

```
@@ -140,11 +140,11 @@ public function testStoreComponentInstancesRoundTripsThroughConsume(): void
         $entry = DeferredRequestRegistry::consume('dr_components');
 
         self::assertNotNull($entry);
-        self::assertCount(2, $entry['components']);
-        self::assertSame('cmp_1', $entry['components'][0]['instance_id']);
-        self::assertSame('ui-playground.leads-grid', $entry['components'][0]['name']);
-        self::assertSame(['limit' => 5], $entry['components'][0]['props']);
-        self::assertSame(['slot-a'], $entry['slots']);
+        self::assertCount(2, $entry->components);
+        self::assertSame('cmp_1', $entry->components[0]['instance_id']);
```

PHPStan (2.2.8)

[error] Call to an undefined static method Semitexa\Ssr\Tests\Unit\Isomorphic\DeferredRequestRegistryTest::assertSame().


(staticMethod.notFound)